packages/frint-component-utils/src/streamProps.js

```diff
@@ -1,12 +1,16 @@
-import _ from 'lodash';
-import { Observable } from 'rxjs';
+import isPlainObject from 'lodash/isPlainObject';
+import { Observable } from 'rxjs/Observable';
+import { of as of$ } from 'rxjs/observable/of';
+import { merge as merge$ } from 'rxjs/observable/merge';
+import { concatMap as concatMap$ } from 'rxjs/operator/concatMap';
+import { scan as scan$ } from 'rxjs/operator/scan';
```

Comment:
`Observable` and its operators (`of`, `merge`, etc) both are imported. Should we either get rid of `Observable` here (adjusting its usage in file) or don't import the operators separately and use them as part of `Observable` in this file?

Reply:
There's a difference to take into account:

```js
// this one comes with all the operators
import { Observable } from 'rxjs';
typeof Observable.prototype.map === 'function';

// this one does not come with any operators
import { Observable } from 'rxjs/Observable';
typeof Observable.prototype.map === 'undefined';
```

That's why when importing any RxJS types individually, we need to import the operators individually too.

Reply:
I see. With this approach the imports and the code itself become more complicated. Also `::` bind operator has to be used everywhere and it's still in `stage-0` actually: https://babeljs.io/docs/plugins/preset-stage-0/

They also suggest following approach to importing individual operators
```
import { Observable } from 'rxjs/Observable';
import 'rxjs/add/observable/of';
import 'rxjs/add/operator/map';

Observable.of(1,2,3).map(x => x + '!!!');
```
(found here https://github.com/ReactiveX/rxjs#es6-via-npm)

The code becomes much cleaner in my opinion, but then you're stepping on a slippery path of indirect imports, which modify other modules at the runtime and you no longer know what depends on what: https://github.com/ReactiveX/rxjs/blob/master/src/add/operator/map.ts

Reminds me of jQuery plugins 😱 

I think the approach you chose is better.

Reply:
By the way, since you went for the bind operator approach, shall we forbid importing `rxjs/add/*` modules as well?

Reply:
good point. I have added that check in the most recent commit.